Exception = Object.new(
  backtrace: []
  message: ""
)
